Source/WebCore/rendering/svg/RenderSVGResourceClipper.h

```diff
@@ -30,6 +30,32 @@ class GraphicsContext;
 class ImageBuffer;
 class SVGClipPathElement;
 
+struct ClipperData {
+    WTF_MAKE_STRUCT_FAST_ALLOCATED;
+
+    struct Inputs {
+        bool operator==(const Inputs& other) const { return objectBoundingBox == other.objectBoundingBox && clippedContentBounds == other.clippedContentBounds && scale == other.scale && effectiveZoom == other.effectiveZoom; }
```

Comment:
Maybe this can be more readable:

```
bool operator==(const Inputs& other) const
{
    return std::tie(objectBoundingBox, clippedContentBounds, scale, effectiveZoom) == std::tie(other.objectBoundingBox, other.clippedContentBounds, other.scale, other.effectiveZoom);
}
```

Reply:
Nice trick!